### ISSUE TITLE
Instructions for installing virtual machine on Windows

### ISSUE THREAD

**Issue by bgiobbe** [railsbridge-boston/railsbridge-boston#94]
I agree with not-repeating-ourselves; however, the instructions on the Virtual Machine page are not beginner-friendly and don't explain where to enter the `vagrant box ...` command.

Custom instructions for each platform might be helpful because the files to download and install differ for each platform as do the command line app for downloading the railsbridgebos.box.

The SSH key, Git config, and Heroku account steps need explicit instructions, too, common to all platforms.

**Comment by bcarreno** [comment]
Go ahead and write the instructions according to what you consider better (I'd put a link to the other repo though).  If we need to DRY them we can do it later.

http://www.railsbridgeboston.org/installfest contains the common instructions.  There's already a section on Heroku that we'd have to update because it relies on RailsInstaller for Windows.  This part also talks about SSH keys.

I'd create a section with git config instructions here http://www.railsbridgeboston.org/installfest

Note that if you want to update https://github.com/railsbridge-boston/railsbridge-virtual-machine you can make another PR.

Thanks for your help!

**Comment by bgiobbe** [comment]
The Mac instructions also use RailsInstaller. Do you intend to use the virtual machine on Mac as well as Windows?

**Comment by bcarreno** [comment]
Good point.  The answer: https://github.com/railsbridge-boston/railsbridge-boston/issues/86#issuecomment-23833107
We already make a distinction http://www.railsbridgeboston.org/installfest/install/osx so it should look like this:

10.6 - 10.8: Go ahead, your instructions are below
10.5 running Intel processor: Go here for your installation instructions using Vagrant
10.5 running non-Intel processor: Go here for your installation instructions using Xcode

I'd leave the Linux instructions unchanged for now.

**Comment by bgiobbe** [comment]
You don't have the students open GitHub accounts in the class? Just Heroku?

**Comment by bcarreno** [comment]
Correct.

**Comment by ltanev** [comment]
where I can find a detail explanation about setting of  Railsbridge Virtual Machine. The explanations are for people that know how to do  it. I am new and I am looking for complete and detail tutorial.
I have windows 8 and I have trouble running Ruby on Rails apps. I was so please when I saw that I have some choice to run my apps, but for me is impossible to understand how to start. I open the git gray cat and on the top of the window says window power shell and you says. You do not use it. How easy would be if you have a picture of the command prompt window. There are many prompt windows when you start to work with ruby on rails.
